src/commands/config/mod.rs

```diff
@@ -17,10 +17,40 @@ pub fn set_file_mode(file: &PathBuf) {
         .expect("could not set permissions on file");
 }
 
-pub fn global_config(email: String, api_key: String) -> Result<(), failure::Error> {
-    let s = GlobalUser { email, api_key };
+pub fn global_config(api_key: bool) -> Result<(), failure::Error> {
+    let mut user = GlobalUser {
+        email: None,
+        api_key: None,
+        api_token: None,
+    };
 
-    let toml = toml::to_string(&s)?;
+    if !api_key {
+        // Default: use API token.
+        message::info("Looking to use a Global API Key and email instead? Run \"wrangler config --api-key\". (Not Recommended)");
+        println!("Enter API token: ");
+        let mut api_token_str: String = read!("{}\n");
```

Comment:
it feels...wrong to put all this interactivity into a constructor, when originally we fed arguments to this function. i think it makes more sense to create a second function for token and call it from main, and to get the arguments from main as well.

Reply:
yeah, that'll be a small fix!